src/solver/types.jl

```diff
@@ -120,7 +120,9 @@ diffuse_momentum(::FriersonDiffusion{DM}) where {DM} = DM
 diffuse_momentum(::Nothing) = false
 
 abstract type AbstractSponge end
+Base.Broadcast.broadcastable(x::AbstractSponge) = tuple(x)
 Base.@kwdef struct ViscousSponge{FT} <: AbstractSponge
+    zmax::FT
```

Comment:
I have the same question that @Sbozzolo has for the Rayleigh sponge: Is it possible to move this out of the struct?

Reply:
Yes, I'll update this to do that